### PR TITLE
Connect Formspree to contact form

### PR DESCRIPTION
**Pull request by codedbystuart**
## Changes
* index.html

## Features
Added functionality to contact form using formspree

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,182 +1,242 @@
...
-
-
+    </section>
 
+    <section class="contact-form">
+      <h4 class="contact-title">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h4>
+      <form action="https://formspree.io/f/xnqrnjll" method="post">
```
Comment (thecodechaser) [inline index.html:191]: - [ ] Please use the correct font weight for the heading and make sure it matches the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.

### Your design

image

### Figma design

image

File: index.html [hunk 2]
```diff
@@ -1,182 +1,242 @@
...
+          id="message"
+          class="form-item"
+          cols="30"
+          rows="10"
+          placeholder="Enter text here"
+          required
+          maxlength="500"
+        ></textarea
+        ><br />
+        <input type="submit" class="submit form-item" value="Get in Touch" />
```
Comment (thecodechaser) [inline index.html:220]: - [ ] Please implement the button interaction for this submit button. (Hover, Active, etc.)
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @codedbystuart, and @Maxwell011👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @codedbystuart & @Maxwell011 ✋



Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**